files/images/teamspeak_viewer/badges/gamescom_2018_played_details.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" width="64" height="64" viewBox="0 0 64 64"><title>BlackOps4_Strapless</title><path d="M63.38,37.69c-2-.73-3-4-3.41-5.63-.06-.3-.12-.59-.19-.89a1.32,1.32,0,0,1,0-.13h0A29.48,29.48,0,0,0,1.65,37.37v17A162.29,162.29,0,0,1,21.76,52c1.26-.06,3.58-4.18,5.72-7.15,3.52-4.89,7.72-9.23,13.75-9q6.21.24,12.14.76Z" style="fill:#7d8466"/><path d="M19.81,10.89c-.5,1.54-.95,3.22-1.37,5a83.92,83.92,0,0,0-2,18.48v18c1.3-.11,2.65-.21,4-.28V34.4A78.76,78.76,0,0,1,22.36,17a46.06,46.06,0,0,1,2.39-7.69A28.64,28.64,0,0,0,19.81,10.89Z" style="fill:#646d4a"/><path d="M12.09,15.92a83.31,83.31,0,0,0-2,18.48V53.08c1.35-.18,2.69-.34,4.05-.48V34.4A78.76,78.76,0,0,1,16,17c.41-1.83.88-3.49,1.37-5a29.69,29.69,0,0,0-5.2,3.52Z" style="fill:#646d4a"/><path d="M3.77,34.4V54c1.39-.22,2.73-.43,4-.62v-19a79.92,79.92,0,0,1,1.6-16.21,29.85,29.85,0,0,0-5.28,8.07C3.9,28.91,3.77,31.62,3.77,34.4Z" style="fill:#646d4a"/><path d="M31.13,8.64A30.29,30.29,0,0,0,26.84,9a52.74,52.74,0,0,0-2,7,83.31,83.31,0,0,0-2,18.48v17a51.14,51.14,0,0,0,4.05-5.62V34.4A78.76,78.76,0,0,1,28.72,17a43.11,43.11,0,0,1,2.69-8.38Z" style="fill:#646d4a"/><path d="M1.65,55.36a1,1,0,0,1-.12-2l21.35-2.7c1.31-3.16,8-17.74,19.49-16.08l20,2.11a1,1,0,1,1-.21,2l-20-2.11C31.05,35,24.61,51.77,24.54,51.94l-.21.57L1.78,55.35Z" style="fill:#4b5135"/></svg>
```

Comment:
Why are there no line breaks here?

Reply:
Rechecked with original source. The file is untouched from original.